src/main/java/com/jcabi/email/Protocol.java

```diff
@@ -42,11 +42,13 @@
  * <pre> Postman postman = new Postman.Default(
  *   new SMTP(
  *     new Token("user", "password").access(
- *       new Protocol.SMTPS("smtp.gmail.com", 587)
+ *       new Protocol.SMTP("bind", "port")
```

Comment:
@amihaiemil but isn't it better to provide real SMTPS gmail address instead of `bind`?